orangecontrib/timeseries/agg_funcs.py

```diff
@@ -105,20 +105,33 @@ def __func__(arr):
         return ' ; '.join(map(str, arr))
 
 
-AGG_FUNCTIONS = [
-    Mean,
-    Sum,
-    Max,
-    Min,
-    Median,
-    Mode,
-    Std_deviation,
-    Variance,
-    Product,
-    Weighted_MA,
-    Exponential_MA,
-    Harmonic_mean,
-    Geometric_mean,
-    Count_nonzero,
-    Count_defined,
-]
+AGG_FUNCTIONS = {
+    'ContinuousVariable': [Mean,
+                           Sum,
+                           Max,
+                           Min,
+                           Median,
+                           Mode,
+                           Std_deviation,
+                           Variance,
+                           Product,
+                           Weighted_MA,
+                           Exponential_MA,
+                           Harmonic_mean,
+                           Geometric_mean,
+                           Count_nonzero,
+                           Count_defined
+                           ],
+    'DiscreteVariable': [Mode,
+                         Count_nonzero,
+                         Count_defined
+                         ],
+    'TimeVariable': [Mean,
+                     Max,
+                     Min,
+                     Median,
+                     Mode,
+                     Count_nonzero,
+                     Count_defined
+                     ]
```

Comment:
What is a variance of time? Currently it is a variance of floats and it is meaningless. It would be interesting as a variance of date - epoch values, then cast to some sensible time value.

Reply:
Not the variance, but standard deviation could make sense, as in "the quarantine will end on April 30, give or take a week".

Reply:
What I am saying is that Variance doesn't work on master. Std deviation does, but what it returns is not correct - it seems like it casts std. to datetime, which is just confusing. These methods should be adapted to datetime.
I am happy to include it, if we figure out how to write it properly.